src/storage-client/src/controller.rs

```diff
@@ -106,6 +106,10 @@ pub enum IntrospectionType {
     SourceStatusHistory,
     ShardMapping,
     StorageHostMetrics,
+
+    // Note that this single-shard introspection source will be changed to per-replica,
```

Comment:
nit: I would maybe not say "once storage and compute are merged" but instead sth like "once we allow running/multiplexing multiple sources on a single cluster"